cellprofiler/modules/loadimages.py

```diff
@@ -436,199 +463,241 @@ def example_path_fn():
-            subfolders containing the images with the run ID
-            (e.g., <i>./2009_10_02/1234/</i>) The following regular expression
-            will capture the plate, well, and site in the fields
-            <i>Date</i> and <i>Run</i>:<br>
-            <table border = "1">
-            <tr><td colspan = "2">.*[\\\/](?P&lt;Date&gt;.*)[\\\\/](?P&lt;Run&gt;.*)$</td></tr>
-            <tr><td>.*[\\\\/]</td><td>Skip characters at the beginning of the pathname until either a slash (/) or
-            backslash (\\) is encountered (depending on the operating system)</td></tr>
-            <tr><td>(?P&lt;Date&gt;</td><td>Name the captured field <i>Date</i></td></tr>
-            <tr><td>.*</td><td>Capture as many characters that follow</td></tr>
-            <tr><td>[\\\\/]</td><td>Discard the slash/backslash character</td></tr>
-            <tr><td>(?P&lt;Run&gt;</td><td>Name the captured field <i>Run</i></td></tr>
-            <tr><td>.*</td><td>Capture as many characters as follow</td></tr>
-            <tr><td>$</td><td>The <i>Run</i> field must be at the end of the path string, i.e., the
-            last folder on the path. This also means that the Date field contains the parent
-            folder of the Date folder.</td></tr>
-            </table></p>"""))
+                doc="""\
+*(Used only if you want to extract metadata from the path)*
+
+Enter the regular expression for extracting the metadata from the
+path. Note that this field is available whether you have selected
+*Text-Regular expressions* to load the files or not.
+
+Clicking the magnifying glass icon to the right will bring up a tool
+that will allow you to check the accuracy of your regular expression.
+The regular expression syntax can be used to name different parts of
+your expression. The syntax *(?<fieldname>expr)* will extract whatever
+matches *expr* and assign it to the image’s *fieldname* measurement.
+
+For instance, a researcher uses folder names with the date and
+subfolders containing the images with the run ID (e.g.,
+*.2009_10_02/1234*) The following regular expression will capture
+the plate, well, and site in the fields *Date* and *Run*:
+
++-----------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| .*[\\\\\\/](?P<Date>.*)[\\\\\\\\/](?P<Run>.*)$      |
++-----------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| .*[\\\\\\\\/]                                       | Skip characters at the beginning of the pathname until either a slash (/) or backslash (\\\\) is encountered (depending on the operating system)                               |
```

Comment:
Please add 4 more spaces in the first column of this line to get the table to format correctly.

Reply:
Also add two spaces to the second column of this line.

Reply:
I think I figured it out but I am having an issue updating LoadImages...